independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2817 @@
+                  "description": "result"
+                },
+                "results": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "Name": {
+                        "type": "string",
+                        "description": "Name"
+                      },
+                      "Details": {
+                        "type": "array",
+                        "items": {
+                          "$ref": "#/definitions/DatasheetConfigurationSummaryModel"
+                        },
+                        "description": "Details"
+                      }
+                    }
+                  },
+                  "description": "results"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get Measure Downloads Available",
+        "description": "Gets dictionary, indexed by the measure download code, of all available measure download codes and descriptions that can be used in the measure download API call.",
+        "operationId": "GetMeasureDownloads",
+        "parameters": [{
+            "name": "Accept",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
+          }
+        ]
+      }
+    },
+    "/measure-downloads1/{measure-download-code}": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "parameters": [{
+            "name": "measure-download-code",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The measure download code.",
+            "x-ms-summary": "The measure download code",
+            "x-ms-url-encoding": "single",
+            "x-ms-dynamic-values": {
+              "operationId": "GetMeasureDownloads",
+              "value-path": "Name",
+              "value-collection": "results",
+              "value-title": "Name",
+              "parameters": {
+                "Accept": "application/json"
+              }
+            }
+          }, {
+            "name": "Accept",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",
+            "x-ms-visibility": "internal"
+          }
+        ],
+        "summary": "Get Measure Download",
+        "description": "Gets a measure download, in XLSX format, for the supplied measure download code.",
+        "operationId": "GetMeasureDownload"
+      }
+    },
+    "/measure-downloads/across-reporting-units/{measure-download-code}": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {}

```

Comment:
@paulculmsee Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.